tui/satellite/password.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
-SATELLITE_PASSWORD=$(whiptail --passwordbox --cancel-button "$CANCEL_BUTTON" --ok-button "$OK_BUTTON" --title "$TITLE_PASSWORD" "$CONTENT_PASSWORD" 25 80 3>&1 1>&2 2>&3)
-export SATELLITE_PASSWORD
+SATELLITE_PASSWORD=$(whiptail --passwordbox --cancel-button "$BACK_BUTTON" --ok-button "$OK_BUTTON" --title "$TITLE_PASSWORD" "$CONTENT_PASSWORD" 25 80 3>&1 1>&2 2>&3)
 
 exit_status=$?
-if [ "$exit_status" -eq 1 ]; then
-  exit 0
+
+if [ $exit_status == 0 ]; then
```

Comment:
The `"` are missing around `$exist_status` and `-eq` should be used instead of `==`.